exercises/concept/salary-calculator/.docs/instructions.md

```diff
@@ -20,25 +20,33 @@ multiplierPerDaysSkipped(daysSkipped);
 
 ## 2. Calculate the bonus for products sold
 
-Implement the `multiplierPerProductsSold` and `bonusForProductSold` methods. The ISC pays ten monetary units for each product sold, but if the employee sold more than twenty products, the multiplier is improved to thirteen. `multiplierPerProductsSold` should decide which multiplier is applied and `bonusForProductSold` should return the total bonus in monetary units.
+Implement the `multiplierPerProductsSold` and `bonusForProductsSold` methods. The ISC pays ten monetary units for each product sold, and if an employee sells twenty products or more, the multiplier improves to thirteen. `multiplierPerProductsSold` should determine which multiplier to apply, and `bonusForProductSold` should return the total bonus in monetary units.
```

Comment:
Reading this instruction I think the `multiplierPerProductsSold` method is poorly named. Usually "per" is followed by a singular noun. Would renaming it to `multiplierForProductsSold` be more appropriate?